css-text-3/Overview.bs

```diff
@@ -1594,7 +1594,7 @@ Text Processing</h3>
 
       <li>A sequence at the end of a line
         (ignoring any intervening <a>inline box</a> boundaries)
-        of <a>collapsible</a> spaces (U+0020)
+        of <a>collapsible</a> [=white space=]
```

Comment:
This needs to say spaces, because line breaks are not necessarily removed.

Reply:
Same question as https://github.com/w3c/csswg-drafts/pull/4269/files#r320544442

Reply:
Same answer